slides.md

```diff
@@ -52,107 +52,442 @@ CSS Class notes for the 1st Semester
 <!--
 The last comment block of each slide will be treated as slide notes. It will be visible and editable in Presenter Mode along with the slide. [Read more in the docs](https://sli.dev/guide/syntax.html#notes)
 -->
----
-hideInToc: true
+
 ---
 
-# Table of contents
+## Table of contents
 
 <Toc columns="2" minDepth="1" maxDepth="2"></Toc>
+
 ---
 
 # Getting Started with CSS?
 
+### CSS which stands in for Cascading Style Sheets is a stylesheet language used to describe the presentation of a document written in HTML or XML.
 
-### CSS which stands in for Cascading Style Sheets is a stylesheet language used to describe the presentation of a document written in HTML or XML. 
 <br/>
 
 ### Just as HTML serves as the skeletal part of the web, CSS describes how the element should be rendered on the web.
 
 ### We use CSS to style our HTML elements and this is what you're going to learn througout this module.
+
 ---
 
-# Selectors and Specificity
+# Selectors
 
+Before we move deeply into Selectors, let's dive into CSS rule which is a block of code, that has one or more selectors and one or more declarations.
 
+<img src="https://web.dev/static/learn/css/selectors/image/an-image-a-css-rule-the-ced38545b4bec.svg"/>
 
 ---
 
-# Box Model
+<ins>Definition of selectors</ins>
+
+Looking at the image in the previous slide we'll notice that CSS selector is the first part of a CSS rule. In order to choose or select HTML elements that's going to carry the CSS property values inside the rule we have to use CSS Selector. In summary, for us to add a style for a particular HTML element we need a selector.
+
+Types of selectors
+
+- Universal selector: This is also know as a wildcard, selects every single element in the document. It is represented by the asterisk character <kbd>\*</kbd>
+
+<ins>Code Example</ins>:
 
+```
```

Comment:
Use ````css` for better formatting

Reply:
I have changed all the formatter to either ```css or ```html for better formatting.